Election 'cycle' text changes to web pages

cptechiegal opened the pull request:
## Summary (required)

- Addresses # 1444 https://app.zenhub.com/workspace/o/18f/fec-cms/issues/1444

- Changes the use of the verbiage 'cycle' on various web pages

## Impacted areas of the application
List general components of the application that this PR will affect:

-  Front end web pages components: 
cycle-select, candidate-cycle-select, election-cycle-select and election-filter

## Changes Completed:
Before screenshots can be seen in issue#1444

- Changed the Candidate Profile pages
After design discussions this month, the final decision was to change the text from: "full cycle"
to: "All Years"
Manual Testing:
a)candidate/financial-summary==>Candidate Profile pages
Presidential
https://www.fec.gov/data/candidate/P60022654/
![election periods - presidential- after 1444 chgs](https://user-images.githubusercontent.com/24944162/33968324-627b2fbc-e035-11e7-97eb-1602b4b16464.jpeg)

Senate
https://www.fec.gov/data/candidate/S6MD03177/
b)Candidate/other-spending-tab
https://www.fec.gov/data/candidate/S6MD03177/?tab=other-spending
c)Candidate/about-candidate
https://www.fec.gov/data/candidate/S6MD03177/?tab=about-candidate

- Changed the Candidates for President data table (Edit Filters section)
pulldown menu label text from: "Full Cycle" to : “All Years”
Manual Testing:
https://www.fec.gov/data/candidates/president/?election_year=2016&cycle=2016&election_full=true
![presidential candidate edit filters](https://user-images.githubusercontent.com/24944162/33968335-6f5a90ba-e035-11e7-8d67-e0072f1f2a61.jpeg)


- Changed the Election lookup pages
Changed the text label above the period from: "Election Cycle" to: "2-year period"
see separate issue --> #1605

- Raising and Spending pages
Changed the label text for the heading on the Elections pulldown menu from ‘Election Cycle’ to ‘Election’
Manual Testing:
https://www.fec.gov/data/candidate/H8WI01024/?tab=raising
https://www.fec.gov/data/candidate/P80001571/?tab=raising
https://www.fec.gov/data/candidate/S2MA00170/?tab=raising
https://www.fec.gov/data/candidate/S2MA00170/?tab=spending
![candidate profile - raising page after 1444](https://user-images.githubusercontent.com/24944162/33968341-77a1ea3e-e035-11e7-8625-74081db08779.jpeg)
![candidate profile - spending page after 1444](https://user-images.githubusercontent.com/24944162/33968345-7c16f910-e035-11e7-8ca2-837af21a32d5.jpeg)


## Related PRs
List related PRs against other branches:
Issue 1605 https://app.zenhub.com/workspace/o/18f/fec-cms/issues/1605
PR 1630 https://app.zenhub.com/workspace/o/18f/fec-cms/issues/1630

lbeaufort commented:
@cptechiegal and @jenniferthibault - I merged https://github.com/18F/fec-cms/pull/1630 which had changed `fec/data/templates/macros/cycle-select.jinja` to "Two-year period" but this one seems to have "Two-year Period." Should these labels capitalize just the first letter of the first word? It looks like others ("All Years") are also capitalizing both words. Thanks!

jenniferthibault commented:
Correct @lbeaufort — we use sentence case capitalization style across the site, so only the first word should be capitalized, even in field labels (except for when it's a proper noun). So it should be  `Two-year period` and `All years`

cptechiegal commented:
Updated the following files to address the PR review comments:
1) fec/data/templates/macros/cycle-select.jinja
macros: committee_cycle_select and election_cycle_select
Restored to same as PR1630: Two-year period
2) fec/static/js/templates/election-cycles.hbs
Changed to: All years
3) fec/fec/static/js/templates/electionCycles.hbs
Changed to: All years
4) fec/fec/tests/js/cycle-select.js
Changed to: All years
